src/plone/rest/service.py

```diff
@@ -1,10 +1,10 @@
 # -*- coding: utf-8 -*-
-from Products.Five import BrowserView
+import zope.publisher.browser
```

Comment:
BrowserView does not have traversal behavior and when its gotten on the BaseRequest, I switched to BrowserPage to avoid problems on traversal. About zope.publisher - Products.Five I could not see any problem on using BrowserPage from zope.publisher, but maybe I'm missing something.